lib/stat_tracker.rb

```diff
@@ -23,6 +23,41 @@ def initialize(files)
     end
   end
 
+  def all_season_game_id(season)
```

Comment:
looks good here, we can combine this same method on mine since we have the same one essentially.